docs/guides/distribution/sign-macos.md

```diff
@@ -13,8 +13,11 @@ If you are not utilizing GitHub Actions to perform builds of OSX DMGs, you will
 
 ## Requirements
 
-- Xcode 11 or above.
-- An Apple Developer account enrolled in the [Apple Developer Program].
+- macOS 10.13.6 or later
+- Xcode 10 or later
+- An Apple Developer account enrolled in the [Apple Developer Program]
+
+The prerequisites above are also the minimum conditions for notarization. For more details please read the developer article on [notarizing macOS software before distribution].
```

Comment:
```suggestion
For more details please read the developer article on [notarizing macOS software before distribution].
```
The first sentence feels redundant because the page itself is about code signing _and_ notarization. Couldn't think of a better phrasing so imo we should just remove it.